Leyline-gg/leyline-discord
# Implement Kind words reaction system

## Summary of Changes
- Completely refactor `ReactionCollector` class, mainting current functionality
    - is now much more modular
- Finishes everything non-xp related to #33 
- Messages sent in kind-words channel can now be approved by mods

## Imagery
None

## Testing Procedure
- Send a message in #kind-words
- Ensure no reactions appear immediately
- As a mod, react with a ✅ or ❌ to the message
- Ensure you receive LLP or that the post is rejected accordingly

## Additional Notes
- Does not include XP awardal, like #35
